includes/subscriptions/class-wc-payments-subscriptions.php

```diff
@@ -67,7 +67,7 @@ public static function init( WC_Payments_API_Client $api_client, WC_Payments_Cus
 		include_once __DIR__ . '/class-wc-payments-subscriptions-empty-state-manager.php';
 
 		self::$product_service      = new WC_Payments_Product_Service( $api_client );
-		self::$order_service        = new WC_Payments_Order_Service( $api_client );
+		self::$order_service        = $order_service;
```

Comment:
For future reference, should we reorder the assignments and add some comments here, to avoid easily following the existing example, and creating new instances?

Looking at it now, I guess reordering the whole method while adding comments would make sense:

1. Store dependencies (in this case just the order service). `// Store dependencies.`
2. Include new files. `// Load services`.
3. `// Instantiate additional classes.`

So the method would look like this:

```php
// Store dependencies.
self::$order_service = $order_service;

// Load Services.
include_once __DIR__ . '/class-wc-payments-product-service.php';
include_once __DIR__ . '/class-wc-payments-invoice-service.php';
include_once __DIR__ . '/class-wc-payments-subscription-service.php';
include_once __DIR__ . '/class-wc-payments-subscription-change-payment-method-handler.php';
include_once __DIR__ . '/class-wc-payments-subscriptions-plugin-notice-manager.php';
include_once __DIR__ . '/class-wc-payments-subscriptions-empty-state-manager.php';
include_once __DIR__ . '/class-wc-payments-subscriptions-event-handler.php';
include_once __DIR__ . '/class-wc-payments-subscriptions-onboarding-handler.php';
include_once __DIR__ . '/class-wc-payments-subscription-minimum-amount-handler.php';

// Instantiate additional classes.
self::$product_service      = new WC_Payments_Product_Service( $api_client );
self::$invoice_service      = new WC_Payments_Invoice_Service( $api_client, self::$product_service, self::$order_service );
self::$subscription_service = new WC_Payments_Subscription_Service( $api_client, $customer_service, self::$product_service, self::$invoice_service );
self::$event_handler        = new WC_Payments_Subscriptions_Event_Handler( self::$invoice_service, self::$subscription_service );

new WC_Payments_Subscription_Change_Payment_Method_Handler();
new WC_Payments_Subscriptions_Plugin_Notice_Manager();
new WC_Payments_Subscriptions_Empty_State_Manager( $account );
new WC_Payments_Subscriptions_Onboarding_Handler( $account );
new WC_Payments_Subscription_Minimum_Amount_Handler( $api_client );
```